lib/multiplexer/proxyline.go

```diff
@@ -85,6 +85,8 @@ var (
 	ErrIncorrectRole = errors.New("could not find required system role on the signing certificate")
 	// ErrNonLocalCluster is returned when we received signed PROXY header, which signing certificate is from remote cluster.
 	ErrNonLocalCluster = errors.New("signing certificate is not signed by local cluster CA")
+	// ErrNoHostCA is returned we CAGetter could not get host CA, it can happen for example if auth server is not running at the moment
```

Comment:
Nit: comment brevity
```suggestion
	// ErrNoHostCA is returned when CAGetter could not get host CA, for example if auth server is not available
```